src/cheri/introduction.adoc

```diff
@@ -30,115 +30,123 @@ be brute forced.
-| Extension                                                 | Status        | Comment
-|<<rv32y,{cheri_base_ext_name}>>                            | Stable        | This extension is a candidate for freezing
-|<<section_rvy_sentry_insn_ext>>                            | Stable        | This extension is a candidate for freezing
-|<<section_cheri_hybrid_ext,{cheri_default_ext_name}>>      | Stable        | This extension is a candidate for freezing
-|<<section_zyseal>>                                         | Experimental  | This extension is not yet a candidate for freezing
-|<<section_xycheriot>>                                      | Experimental  | This extension is not yet a candidate for freezing
+| Extension                                                 | Description
+|<<rv32y,{cheri_base_ext_name}>>                            | Base ISA
+|<<app_cap_description,{rvy_default_encoding_name}>>        | Default capability encoding
+|<<section_rvy_sentry_insn_ext>>                            | Sentry extension for CFI
+|<<section_cheri_hybrid_ext,{cheri_default_ext_name}>>      | Hybrid extension for RVI compatibility
+|<<rvy_zca_insn_table,   {rvy_zca_ext_name}>>               | New 16-bit encodings added to Zca
+|<<rvy_zba_insn_table,   {rvy_zba_ext_name}>>               | New 32-bit encodings added to Zba
+|<<rvy_zalrsc_insn_table,{rvy_zalrsc_ext_name}>>            | New 32-bit encodings added to Zalrsc
+|<<rvy_zaamo_insn_table, {rvy_zaamo_ext_name}>>             | New 32-bit encodings added to Zaamo
 |=============================================================================================================================================================
 
-.Other unprivileged extension status and summary (used by CHERI software), status and summary
+.Unprivileged stable extensions where {cheri_base_ext_name} modifies the behavior
+[#unpriv-mod-extension-status,reftext="Extension Status and Summary"]
+[options=header,align=center,width="90%"]
+|=============================================================================================================================================================
+| Extension                                                 | Description
+|<<{rvy_i_mod_file_name},     {rvy_i_mod_ext_name}>>        | RVI instructions modified by {cheri_base_ext_name}
+|<<{rvy_zca_mod_file_name},   {rvy_zca_mod_ext_name}>>      | Zca instructions modified by {cheri_base_ext_name}
+|<<{rvy_zicsr_mod_file_name}, {rvy_zicsr_mod_ext_name}>>    | Zicsr instructions modified by {cheri_base_ext_name}
+|<<{rvy_zicbom_mod_file_name},{rvy_zicbom_mod_ext_name}>>   | Zicbom instructions modified by {cheri_base_ext_name}
+|<<{rvy_zicboz_mod_file_name},{rvy_zicboz_mod_ext_name}>>   | Zicboz instructions modified by {cheri_base_ext_name}
+|<<{rvy_zicbop_mod_file_name},{rvy_zicbop_mod_ext_name}>>   | Zicbop instructions modified by {cheri_base_ext_name}
```

Comment:
Should we add V here?

Reply:
V comes later as I've classified it as "experimental", also H